modules/hosted-control-planes-monitoring-dashboard.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-managing.adoc
+
+:_content-type: PROCEDURE
+[id="hosted-control-planes-monitoring-dashboard_{context}"]
+= Creating monitoring dashboards for hosted clusters
+
+The HyperShift Operator can create or delete monitoring dashboards in the management cluster for each hosted cluster that it manages. You can enable this functionality when you install the HyperShift Operator.
+
+//lahinson - oct. 2023 - do users still need to enable the HyperShift Operator, or is it enabled by default for OCP 4.14 since the hosted control planes feature is enabled by default?
```

Comment:
@muraee Can you answer my question in the comment on line 11? Thanks!

Reply:
sorry, I don't have an answer for this.

Reply:
@muraee No worries. I asked the wider team to see if anyone knows.